yapio.c

```diff
@@ -1102,15 +1114,20 @@ yapio_open_fpp_file(int rank, int oflags)
 static void
 yapio_setup_test_file(const yapio_test_group_t *ytg)
 {
+    int path_len;
     yapio_verify_test_directory();
 
-    int path_len = snprintf(yapioTestFileName, PATH_MAX, "%s/%s%d_%s",
+    if (yapioInitFromMdFile)
+        path_len = strlen(yapioTestFileName);
+    else
+        path_len = snprintf(yapioTestFileName, PATH_MAX, "%s/%s%d_%s",
```

Comment:
i know it is legacy code but later you should check for -1 if the output is truncated

Reply:
done